### ISSUE TITLE
Does this plugin actually prevent closing pinned tabs?

### ISSUE THREAD

**Issue by martinjo** [hdykokd/obsidian-advanced-close-tab#6]
My pinned tab still gets closed (CMD+W). If it only adds a new command to close unpinned tabs I think the description could a bit clearer :)

**Comment by hdykokd** [comment]
@martinjo

Do you assign the plugin's Close current tab command to hotkey?
The plugin does not change the behavior of the native command, but provides a new command.

**Comment by hdykokd** [comment]
>  If it only adds a new command to close unpinned tabs I think the description could a bit clearer :)

Oh, I had overlooked it. (I was sick and not paying enough attention.)
Exactly, I will update the description.

How about this?

an Obsidian plugin that provides commands to close only unpinned tabs.

I am not a native English speaker, so I am not sure if the nuances are correct.

**Comment by StevePhuc** [comment]
yes please add it to the description . it is not clear need to read this discussion to find out. 

BTW thank for the plugin and maybe soon publish to the store

**Comment by devklepacki** [comment]
Come on this wording is terrible. Obsidian's pinned tabs behavior is also terrible, but framing this plugin as "An Obsidian plugin to prevent closing pinned tab." which it does not achieve at all, is a sad joke for someone who have just spent 15 minutes figuring out why it's not working. This should really be changed to not mislead the very few people who need such a thing 🙂

**Comment by hdykokd** [comment]
Sorry for the confusion. I no longer use Obsidian and had forgotten about this plugin. I will archive it.
